kit.yml

```diff
@@ -73,4 +73,4 @@ provided:
         key:         { multiline: true }
     mgmt/certs/ca:
       keys:
-        certificate: { multiline: true }
\ No newline at end of file
+        certificate: { multiline: true }
```

Comment:
This diverged from master, which is my fault. Marking this as it... uh... needs to be re-removed.